sdk/src/manifest.rs

```diff
@@ -735,6 +735,26 @@ impl Manifest {
         store.save_to_asset(source_path.as_ref(), signer, dest_path.as_ref())
     }
 
+    /// Embed a signed manifest into a stream using a supplied signer.
+    ///
+    #[cfg(feature = "sign")]
+    pub fn embed_stream(
+        &mut self,
+        format: &str,
+        stream: &mut dyn CAIReadWrite,
+        signer: &dyn Signer,
+    ) -> Result<Vec<u8>> {
```

Comment:
What is the content of the `Vec<u8>` in the success case?

Reply:
Need to document that, it is the embedded manifest. It could be useful for uploading to a cloud.